installer_test.go

```diff
@@ -98,3 +99,49 @@ func TestInstaller_Install(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestInstaller_Install_enterprise(t *testing.T) {
+	testutil.EndToEndTest(t)
+
+	// most of this logic is already tested within individual packages
+	// so this is just a simple E2E test to ensure the public API
+	// also works and continues working
+
+	tmpBinaryDir := t.TempDir()
+	tmpLicenseDir := t.TempDir()
+
+	i := install.NewInstaller()
+	i.SetLogger(testutil.TestLogger())
+	ctx := context.Background()
+	_, err := i.Install(ctx, []src.Installable{
+		&releases.ExactVersion{
+			Product:    product.Vault,
+			Version:    version.Must(version.NewVersion("1.9.8")),
+			InstallDir: tmpBinaryDir,
+			Enterprise: releases.EnterpriseOptions{
+				Enterprise: true,
+				LicenseDir: tmpLicenseDir,
+			},
+		},
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Ensure the binary was installed
+	if _, err = os.Stat(filepath.Join(tmpBinaryDir, "vault")); err != nil {
```

Comment:
This test currently fails on Windows, we can get around that e.g. with

```go
binName := "vault"
if runtime.GOOS == "windows" {
	binName = "vault.exe"
}
```

Reply:
Ah, thank you!